soh/soh/Enhancements/enemyrandomizer.cpp

```diff
@@ -12,6 +12,34 @@ extern "C" {
 
 extern "C" uint32_t ResourceMgr_IsSceneMasterQuest(s16 sceneNum);
 
+const char* enemyCVarList[] = {
+    CVAR_ENHANCEMENT("Enemies.Keese"),          CVAR_ENHANCEMENT("Enemies.FireKeese"),
```

Comment:
I think it would be better to have them be under "Enemizer" or "EnemyRando" instead of "Enemies", as "Enemies" is a bit vague.

Reply:
Happy to adjust as needed. My initial thought was to follow CVAR_ENHANCEMENT("RandomizedEnemies") so I used "Enemies" as the sub.

Let me know either way and I'll fix it.

Reply:
Right, but "RandomizedEnemies" is more specific XD. Maybe "RandomizedEnemyList"?

Reply:
Updated <3

Reply:
You didn't change the ImGui stuff XD

Reply:
what do you mean? It says I did ;P